lib/WorseReflection/Bridge/TolerantParser/Diagnostics/DocblockMissingExtendsTagProvider.php

```diff
@@ -184,7 +184,7 @@ class NeedGeneric
                 }
 
                 /**
-                 * @extends NeedGeneic<int>
+                 * @extends NeedGeneric<int>
```

Comment:
Does this class also exist? Can't find that in the list of changed files.

Reply:
I am not able to find it.
https://github.com/search?q=repo%3Aphpactor%2Fphpactor%20NeedGeneic&type=code

Reply:
On the next line it says `class Foobar extends NeedGeneric`

Reply:
I see!!!
`MissingExtendsTag` so it must not match the extended class.

Reply:
Thank you @mamazu!